docs/setup/aks/getting-started-aks.md

```diff
@@ -19,13 +19,13 @@ Follow this step-by-step guide to set up a Nuclio development environment that u
 
 Before starting the set-up procedure, ensure that the following prerequisites are met:
 
-- You have an Azure account. If you don't have an account, you can [create one for free](https://azure.microsoft.com/free/).
+- You have an Azure account. If you don't have an account, you can [create one for free](https://azure.microsoft.com/en-us/free/).
 - The [Azure CLI](https://docs.microsoft.com/cli/azure/) (`az`) is installed on your installation machine.
     See the [Azure CLI installation guide](https://docs.microsoft.com/cli/azure/install-azure-cli).
 
 ## Set up your AKS cluster
 
-1.  <a id="create-resource-group"></a>**Create a resource group** by running the following `az` command (see the [Azure CLI documentation](https://docs.microsoft.com/cli/azure/group#az_group_create)):
+1.  <a id="create-resource-group"></a>**Create a resource group** by running the following `az` command (see the [Azure CLI documentation](https://learn.microsoft.com/en-gb/cli/azure/group?view=azure-cli-latest#az-group-create)):
```

Comment:
Not super critical, but align all of the links in this doc to the same language (there are more `eb-gb` below).
```suggestion
1.  <a id="create-resource-group"></a>**Create a resource group** by running the following `az` command (see the [Azure CLI documentation](https://learn.microsoft.com/en-us/cli/azure/group?view=azure-cli-latest#az-group-create)):
```

Reply:
Fixed 👍